src/automations/checkInactivity.js

```diff
@@ -98,25 +99,45 @@ async function checkInactivePullRequest(pullRequest) {
   }
 }
 
-async function scrapeInactiveIssues(references, issues) {
-  const ms = this.cfg.activity.check.limit * 86400000;
-  const ims = this.cfg.activity.check.reminder * 86400000;
+async function extractPRLabels(client, repoOwner, repoName, number) {
+  const labels = await client.issues.getIssueLabels({
+    owner: repoOwner, repo: repoName, number: number
+  });
+  const inactiveLabel = labels.data.find(l => {
+    return l.name === client.cfg.activity.inactive;
+  });
+  const reviewedLabel = labels.data.find(l => {
+    return l.name === client.cfg.activity.pullRequests.reviewed.label;
+  });
+
+  return {
+    inactiveLabel: inactiveLabel,
+    reviewedLabel: reviewedLabel
+  };
+}
+
+async function scrapeInactiveIssues(client, references, issues) {
```

Comment:
`client` shouldn't be here, it's already been passed on previously.

Reply:
Yes, I had already made the exact same changes you suggested in all the above comments and committed them. I just wanted to know how `Function.apply()` and `Function.call()` would function differently

Reply:
`Function.apply()` and `Function.call()` have the same function, just different syntax :)